slack-lint-checks/src/test/java/slack/lint/retrofit/RetrofitUsageDetectorTest.kt

```diff
@@ -4,10 +4,28 @@ package slack.lint.retrofit
 
 import com.android.tools.lint.detector.api.Detector
 import org.junit.Test
+import retrofit2.http.GET
 import slack.lint.BaseSlackLintTest
 
 class RetrofitUsageDetectorTest : BaseSlackLintTest() {
 
+  private companion object {
+    private val allowUnitResponse =
```

Comment:
I followed other examples of how to make the test recognise this import. Don't know if there is a better way.